tests/unit/backend/chalice/api_server/test_v1_collection.py

```diff
@@ -427,8 +430,143 @@ def test__list_collection__check_owner(self):
             self.assertIn(private_owned, ids)
             self.assertNotIn(private_not_owned, ids)
 
-    def test__delete_collection__OK(self):
-        path = furl(path="/dp/v1/collections/test_collection_id")
+
+class TestCollectionDeletion(BaseAuthAPITest, GenerateDataMixin):
```

Comment:
`BaseAuthAPITest`  includes `GenerateDataMixin` 
```suggestion
class TestCollectionDeletion(BaseAuthAPITest):
```